src/sentry/integrations/slack/webhooks/base.py

```diff
@@ -32,8 +36,8 @@
 class SlackDMEndpoint(Endpoint, abc.ABC):
     slack_request_class = SlackDMRequest
 
-    _METRICS_SUCCESS_KEY = "sentry.integrations.slack.dm_endpoint.success."
-    _METRIC_FAILURE_KEY = "sentry.integrations.slack.dm_endpoint.failure."
+    _METRICS_SUCCESS_KEY = SLACK_WEBHOOK_DM_ENDPOINT_SUCCESS_DATADOG_METRIC
+    _METRIC_FAILURE_KEY = SLACK_WEBHOOK_DM_ENDPOINT_FAILURE_DATADOG_METRIC
```

Comment:
if we're already importing these, do we need to use another variable for them?

Reply:
i keep it cause the name is rather long and often makes 1 line stuff multiple lines after we auto format it.

Reply:
ok